### PR TITLE
Change BlsPriKey to DummyKey

### PR DESCRIPTION
**Pull request by rlan35**

### CODE REVIEW

File: core/resharding.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ const (
 	// FirstEpoch is the number of the first epoch.
 	FirstEpoch = 1
 	// GenesisShardNum is the number of shard at genesis
-	GenesisShardNum = 4
+	GenesisShardNum = 1
```
Comment (mikedoan) [inline core/resharding.go]: this is for debugging only?

File: cmd/harmony/main.go [hunk 2]
```diff
@@ -253,9 +253,9 @@ func createGlobalConfig() *nodeconfig.ConfigType {
 	consensusPriKey := &bls.SecretKey{}
 
 	if *isGenesis {
-		err := consensusPriKey.DeserializeHexStr(genesisAccount.BlsPriKey)
+		err := consensusPriKey.DeserializeHexStr(genesisAccount.DummyKey)
```
Comment (mikedoan) [inline cmd/harmony/main.go]: why using this DymmyKey? BlsPriKey is also generated keys by us currently?

File: core/resharding.go (hunk 1 above)
Comment (rlan35) [reply core/resharding.go]: Sorry my bad. Reverted this line.

File: cmd/harmony/main.go (hunk 2 above)
Comment (rlan35) [reply cmd/harmony/main.go]: To avoid people's confusion.

**Pull request merged**